### DISCUSSION TITLE
Is there any way to change the --yes from y/N to Y/N?

### DISCUSSION THREAD

**Discussion by FriendlyPasser** [pallets/click#2334] [Q&A] [answered]
Is there any way to change the --yes from y/N to Y/N?

**Comment by ThiefMaster** [comment, marked as answer]
What do you mean? `y/N` usually indicates that N is the default if you do not enter anything. If you set a True default it will be `Y/n`` instead.

**Reply by FriendlyPasser** [reply]
i got it . what i mean is to make it the same style. if you have any idea, please let me know.
and could you please tell me how to change the default?

thanks

**Reply by ThiefMaster** [reply]
that's not possible because it makes no sense here - as long as the there is a default value they should not be the same style!

**Reply by FriendlyPasser** [reply]
thank you very much
